examples/standalone/02_hello_window/src/main.rs

```diff
@@ -0,0 +1,169 @@
+use std::sync::Arc;
+
+use winit::application::ApplicationHandler;
+use winit::event::WindowEvent;
+use winit::event_loop::{ActiveEventLoop, ControlFlow, EventLoop};
+use winit::window::{Window, WindowId};
+
+struct State<'a> {
+    surface: wgpu::Surface<'a>,
```

Comment:
Because `Arc` is being used for the window, this lifetime can be removed. It’s currently unconstrained, so it's not doing any technical harm, but it’s superfluous, and should be removed to keep the code simple and clear, and emphasize that unlike the common mistake of passing `&Window`, this is *not* building a self-referential struct. Many people are making this mistake, and they need to be taught about how to avoid it.

```suggestion
struct State {
    surface: wgpu::Surface<'static>,
```

Reply:
@kpreid Thank you for pointing this out, I still get a little confused on when to use `'static` lifetime so this is really appreciated!